napari/_qt/layers/qt_image_layer.py

```diff
@@ -46,22 +47,36 @@ def __init__(self, layer):
         sld.setSingleStep(1)
         sld.setValue(self.layer.iso_threshold * 100)
         sld.valueChanged[int].connect(
-            lambda value=sld: self.changeIsoTheshold(value)
+            lambda value=sld: self.changeisoThreshold(value)
```

Comment:
while you're at it, is there any reason not to just make it this?
```python
sld.valueChanged.connect(self.changeisoThreshold)
```

Reply:
as noted below will defer for followup PR